modules/ROOT/partials/configuration/sandbox_iframes_exclusions.adoc

```diff
@@ -0,0 +1,35 @@
+[[sandbox-iframes-exclusions]]
+== `sandbox_iframes_exclusions`
+
+This option holds a list of URL host names to be excluded from iframe sandboxing when sandbox_iframes is set to true.
+
+[NOTE]
+`sandbox_iframes` option is required to be set in the editor configuration for this option to affect the  is not enabled, this option will have no effect.
```

Comment:
```suggestion
This option allows you to specify a list of domains to exclude from having the `sandbox=""` attribute applied, when the `sandbox_iframes` option is enabled. This option takes an array of strings, each denoting at least the top and second level of a domain to be excluded. By default this option is set to an array of domains which are provided in embed code by popular websites. To enable [sandboxing](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox) on all iframes, set this option to an empty array `[]`.

[NOTE]
When the `sandbox_iframes` option is disabled, all domains will be excluded from sandboxing.
```

Reply:
thank you for updating this, i wasn't sure how to put this one together.